src/Markdig.Wpf.SampleApp/MainWindow.xaml

```diff
@@ -10,6 +10,6 @@
         <CommandBinding Command="{x:Static markdig:Commands.Hyperlink}" Executed="OpenHyperlink" />
     </FrameworkElement.CommandBindings>
     <Grid>
-        <FlowDocumentScrollViewer x:Name="Viewer"/>
+        <markdig:MarkdownViewer Name="Viewer"/>
```

Comment:
Personally I prefer to keep the `x` prefix to show explicitly that `Name` is a [directive](https://docs.microsoft.com/en-us/dotnet/framework/xaml-services/x-name-directive). Since some types might not derive from `FrameworkElement` and I don't want to ask myself all the time I prefer a consistent usage.
But that's more of a personal style preference. No need to worry, I just wanted to point it out. :)